src/Lykke.Logs/LogLevels.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Lykke.Logs
+{
+    internal static class LogLevels
+    {
+        public static IReadOnlyCollection<Microsoft.Extensions.Logging.LogLevel> All { get; }
```

Comment:
Could be internal

Reply:
This is hollywar subject :). As we discussed in Telegram, I leave all similar stuff as is